codalab/worker/main.py

```diff
@@ -252,6 +263,25 @@ def main():
             args.max_work_dir_size,
             args.download_dependencies_max_retries,
         )
+
+    if args.container_runtime == "singularity":
+        singularity_folder = os.path.join(args.work_dir, 'codalab_singularity_images')
+        if not os.path.exists(singularity_folder):
+            logger.info(
+                'Local singularity image location %s doesn\'t exist, creating.', singularity_folder,
+            )
+            os.makedirs(singularity_folder, 0o770)
+        image_manager = SingularityImageManager(
+            args.max_image_size, args.max_image_cache_size, singularity_folder,
+        )
+        docker_runtime = None
```

Comment:
put a TODO comment that singularity workers don't currently work and should be implemented in the future